docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,187 @@
+  - the default value of swh_type is OTHER.  If the swh_type can be identified, the value will be changed in the logic below: `swh_type = "OTHER"`
+  - if the tank_type is CONSUMER_INSTANTANEOUS, COMMERCIAL_INSTANTANEOUS, or RESIDENTIAL_DUTY_COMMERCIAL_INSTANTANEOUS, set the swh_type to INSTANTANEOUS and go to rule assertion: `if tank_type in ["CONSUMER_INSTANTANEOUS", "COMMERCIAL_INSTANTANEOUS", "RESIDENTIAL_DUTY_COMMERCIAL_INSTANTANEOUS"]: swh_type = "INSTANTANEOUS"`; GO TO RULE_ASSERTION
+
+  - get the fuel type of the equipment: `fuel_type = shw_eq.heater_fuel_type`
+  - if the tank_type is CONSUMER_STORAGE or COMMERCIAL_STORAGE, set the swh_type to STORAGE for the fuel type: `if tank_type in ["CONSUMER_STORAGE", "COMMERCIAL_STORAGE"]:`
+    - set the swh_type based on tank_type and fuel type: `if fuel_type == "ELECTRICITY": swh_type = "ELECTRIC_RESISTANCE_STORAGE`
+    - `elif fuel_type == "NATURAL_GAS": swh_type = "GAS_STORAGE`
+    - if the fuel type is propane, this is treated as natural gas in Table 7.8: `elif fuel_type == "PROPANE": swh_type = "GAS_STORAGE"`
+    - if the fuel type is not Electricity, Natural Gas, or Propane, this is not a valid baseline SWH system.  Create a rule note and go directly to the rule assertion: `else: rule_note += "Fuel type: " + fuel_type + " is not a valid fuel type for a service water heating baseline system.  According to ASHRAE 90.1 Table G3.1.1-2 service water heating equipment shall be either electric resistance or natural gas.  According to ASHRAE 90.1 Table G3.1 #11 h, in cases where natural gas is specified as the baseline system, but there is no natural gas available on site, a propane system may be modeled.  "; rule_status = FAIL; GO TO RULE ASSERTION `
+  - if the draw_pattern is given, get the draw pattern: `if shw_eq.draw_pattern != null: draw_pattern = shw_eq.draw_pattern`
+  - otherwise, determine the draw pattern based off the first_hour_rating of the shw_eq: `else:`
+    - First get the first hour rating (gallons): `first_hour_rating = shw_eq.first_hour_rating`
+    - if the first_hour_rating is less than 18 gallons, draw_pattern is "VERY SMALL": `if first_hour_rating < 18: draw_pattern = "VERY_SMALL"`
+    - otherwise if the first_hour_rating is less than 51 gallons, draw pattern is LOW: `elif first_hour_rating < 51: draw_pattern = "LOW"`
+    - otherwise if the first_hour_rating is less than 75 gallons, draw pattern is MEDIUM: `elif first_hour_rating < 75: draw_pattern = "MEDIUM"`
+    - otherwise draw pattern is HIGH: `else: draw_pattern = "HIGH"`
+  - get the capacity of the SHW system (make sure to convert to btu/hr): `input_capacity = swh_eq.input_power`
+  - get the volume of the tank (make sure to convert to gallons): `storage_volume = swh_eq.distribution_system.tank.storage_capacity`
+  - calculate the capacity_per_volume based on the input_capacity and the storage_volume: `capacity_per_volume = shw_eq.rated_capacity / storage_volume`
```

Comment:
english says input_capacity, python says rated_capacity. I think it should be input capacity but I am not positive